src/walletConnect/index.tsx

```diff
@@ -502,7 +504,9 @@ export async function onSessionProposal(
           );
 
           // we only support EVM chains rn
-          const requiredNamespace = requiredNamespaces.eip155;
+          const supportedEvents =
+            requiredNamespaces?.eip155?.events || SUPPORTED_SESSION_EVENTS;
```

Comment:
with the new WC usage this should include `optionalNamespaces` as the first value

Reply:
we will revisit the other session events and confirm we support them (as they often show up in the optionalNamespaces) and then replace this with the full SUPPORTED_SESSION_EVENTS list - will be in a separate PR